cmd/syncIndexer.go

```diff
@@ -66,8 +65,16 @@ func newfolderIndexer() *folderIndexer {
 // storedObjectSize function calculates the size of given stored object.
 func storedObjectSize(so StoredObject) int64 {
 
-	return int64(unsafe.Sizeof(StoredObject{})) + int64(len(so.name)+len(so.relativePath)+len(so.contentDisposition)+
-		len(so.cacheControl)+len(so.contentLanguage)+len(so.contentEncoding)+len(so.contentType)+len(so.ContainerName)+len(so.DstContainerName))
+	return int64(unsafe.Sizeof(StoredObject{})) +
+		int64(len(so.name)+
+			len(so.relativePath)+
+			len(so.contentDisposition)+
+			len(so.cacheControl)+
+			len(so.contentLanguage)+
+			len(so.contentEncoding)+
+			len(so.contentType)+
+			len(so.ContainerName)+
+			len(so.DstContainerName))
```

Comment:
I see that we still have some non-inline members that we need to include in this list.
Pls audit.
Also add a comment above StoredObject struct

// Note: If you make any changes made to this struct, make sure storedObjectSize() in syncIndexer.go is updated accordingly.